### ISSUE TITLE
[Research] Accessibility: Create Cookbook version with scalable font sizes

### ISSUE THREAD

**Issue by jkaltoft** [kirbydesign/designsystem#3568]
Goal: Ensure that text can be resized (see e.g. https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html)

Create a version of the cookbook using scalable font sizes for the UX/design people. The main purpose is to provide a "playground" for UX/design that can be the basis for discussion, questions and further investigation. E.g. should layout also scale with base font size?

This issue is for investigation only. No PR is expected. Delete branch when next step has been decided (depending on UX/design needs)

- [x] Change font size tokens - use relative units
- [x] Change line height tokens - must be unitless
- [x] Adjust typography as needed - there should be no visual changes when user agent font size is unchanged (equiv. to 16px)
- [x] Disable failing tests - or update them if it's feasible within scope
- [x] Take notes on what needs to change - e.g. the way the line-height map and function are used do not work with relative units
- [x] Create new issues based on the notes below

## Notes for "actual implementation"

Read more about lenght units: [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/length) and [W3C](https://www.w3.org/TR/css-values-4/#lengths).

- [x] #3577
    - See [Chromatic build 253](https://www.chromatic.com/build?appId=65ef73cd884a4ee74ff05d78&number=253)
    - May be due to changes in font size or line height tokens?
- [x] #3578
    - We will probably use  `rem` and `em`  for typography (for the most part)
    - Update design tokens
    - Ad hoc font size declarations (if any)
- [x] #3579
    - Line height declaration values should be unitless
    - Line height is a ratio relative to a given element's font size
    - We should be able to create the system's typography without many `line-height`-declarations
    - Ideally the UX/design people should decide on and define a limited number of font size/line height ratios
- [x] #3580
    - May require rethinking how typography should be tested in general
- [x] #3581
    - Create solution that does not depend on pixel units for line height
- [x] #3582
    - Declare `margin-block` values using relative length units instead of pixels
    - Are there other "typography margins" that needs refactoring?
- [x] #3583
    - May require rethinking how and where these design tokens are applied
    - Other useful units we should consider include `dvw`, `cqw`, `ch` and `lh` to name a few, but there are *many* other units. Use what's appropriate.
    - Some values should still be declared using pixels, e.g. breakpoints
    - May include how layout is declared

**Comment by jkaltoft** [comment]
I have created a tracking issue for solving the tasks created during this research: https://github.com/kirbydesign/designsystem/issues/3586
